### ISSUE TITLE
The mod causes things to fly around and errors

### ISSUE THREAD

**Issue by Jennifercutixx** [pWn3d1337/Techguns2#225]
I have been testing my mods because of an issue which is this
![f](https://user-images.githubusercontent.com/60190810/89111660-74300e00-d450-11ea-9c76-69c1d071d0b8.png)
and i found that with just tech guns installed i get the issue in ice biomes. If this could be fixed bc i love the mod just that issues is troubling. thank you and the is amazing. keep the good modding up ^^

**Comment by Jennifercutixx** [comment]
Update: Found out that i loaded a world that still had ice and fire contents in the world and mad ethat come up. in further testing i found the its a conflict with ice and fire's ice villages. When i go near an ice and fire, ice village it give a stack overflow error. I am guessing here but i think its trying to make the ice and fire villagers into the military mobs and cant load the line and gives out an overflow.
In conclusion disabling ice villages in the ice and fire config fixs the stack overflow issue. Ill be sure to update in a new thread if any more issues occour or continue this thread to stop issue spam.

**Comment by Minecr4ck** [comment]
I think you have an option to Close this issue if it was solved.

**Comment by Jennifercutixx** [comment]
it wasnt solved just a solution to the issue don't know if others get this issue.

**Comment by Shibva** [comment]
oh thank god, I though I was the onnly one that was having this this the lattest version as for august 2020

**Comment by Shibva** [comment]
same issue with ice and fire villagers, Can some one please fix that? @Jennifercutixx do you still have a log of that interaction, I thin that could get this fixxed once and for all then

**Comment by Shibva** [comment]
hang On, if I can reproduce with a small number of mods I think that this will make a fix come sooner

**Comment by Shibva** [comment]
![image](https://user-images.githubusercontent.com/38770780/103474944-a4206c00-4d76-11eb-8479-4ced9b905dae.png)
Got it, and its under 30 mods too

**Comment by Shibva** [comment]
ive narrowed it down, the FIsherman Snow villager seems to be causing this mess

I think it might have todo with how its holding its spear or something, idk

**Comment by Shibva** [comment]
![image](https://user-images.githubusercontent.com/38770780/103475031-af27cc00-4d77-11eb-8152-09a7686dbb70.png)
Fisherman is what Im looking at, and thats the hitbox for it over there...


from the log I see this message repeating

[03:57:13] [Client thread/ERROR]: ########## GL ERROR ##########
[03:57:13] [Client thread/ERROR]: @ Post render
[03:57:13] [Client thread/ERROR]: 1283: Stack overflow
[03:57:13] [Client thread/ERROR]: Couldn't render entity
java.lang.NullPointerException: null
	at techguns.client.render.ItemRenderHack.shouldRenderItem(ItemRenderHack.java:124) ~[ItemRenderHack.class:?]
	at techguns.client.render.ItemRenderHack.renderItem(ItemRenderHack.java:54) ~[ItemRenderHack.class:?]
	at net.minecraft.client.renderer.RenderItem.func_181564_a(RenderItem.java:238) ~[bzw.class:?]
	at codechicken.lib.render.item.CCRenderItem.func_181564_a(CCRenderItem.java:293) ~[CCRenderItem.class:?]
	at com.github.alexthe666.iceandfire.client.render.entity.RenderSnowVillager.func_77041_b(RenderSnowVillager.java:30) ~[RenderSnowVillager.class:?]
	at com.github.alexthe666.iceandfire.client.render.entity.RenderSnowVillager.func_77041_b(RenderSnowVillager.java:15) ~[RenderSnowVillager.class:?]
	at net.minecraft.client.renderer.entity.RenderLivingBase.func_188322_c(RenderLivingBase.java:210) ~[caa.class:?]
	at net.minecraft.client.renderer.entity.RenderLivingBase.func_76986_a(RenderLivingBase.java:123) [caa.class:?]
	at net.minecraft.client.renderer.entity.RenderLiving.func_76986_a(SourceFile:41) [caf.class:?]
	at net.minecraft.client.renderer.entity.RenderLiving.func_76986_a(SourceFile:16) [caf.class:?]
	at net.minecraft.client.renderer.entity.RenderManager.func_188391_a(RenderManager.java:384) [bzf.class:?]
	at net.minecraft.client.renderer.entity.RenderManager.func_188388_a(RenderManager.java:368) [bzf.class:?]
	at net.minecraft.client.renderer.RenderGlobal.func_180446_a(RenderGlobal.java:628) [buy.class:?]
	at net.minecraft.client.renderer.EntityRenderer.func_175068_a(EntityRenderer.java:1347) [buq.class:?]
	at net.minecraft.client.renderer.EntityRenderer.func_78471_a(EntityRenderer.java:1259) [buq.class:?]
	at net.minecraft.client.renderer.EntityRenderer.func_181560_a(EntityRenderer.java:1062) [buq.class:?]
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1119) [bib.class:?]
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:398) [bib.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:123) [Main.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_51]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_51]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_51]
	at java.lang.reflect.Method.invoke(Method.java:497) ~[?:1.8.0_51]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]

**Comment by Shibva** [comment]
@pWn3d1337 I found the issue, do You know whats going on here?

**Comment by Shibva** [comment]
using barriers I can confirm that its trying to force it into something else
![image](https://user-images.githubusercontent.com/38770780/103488760-8a663f80-4ddd-11eb-9d69-1834ddea2ead.png)

**Comment by Shibva** [comment]
ive also narrowed it down, its present when Ice and Fire and Techguns (version 3.2) is pressent

Hopeful its a simple fix
